content/applications/sales/subscriptions/reports.rst

```diff
@@ -1,120 +1,384 @@
+When the :guilabel:`Measures` drop-down menu on the :guilabel:`Retention Analysis` reporting page is
+clicked, a series of metric-related options become available.
+
+.. image:: reports/subscriptions-retention-analysis-measures.png
+  :align: center
+  :alt: The measures drop-down menu of the Retention Analysis page.
+
+The metric-related options in the :guilabel:`Measures` drop-down menu on the :guilabel:`Retention
+Analysis` reporting page are:
+
+- :guilabel:`Amount to invoice`
+- :guilabel:`Margin`
+- :guilabel:`Margin (%)`
+- :guilabel:`Prepayment percentage`
+- :guilabel:`Shipping Weight`
+- :guilabel:`Unpaid Amount`
+- :guilabel:`Count`
+
+.. note::
+  The :guilabel:`Count` measure option is selected by default.
+
+To the right of the :guilabel:`Measures` drop-down menu on the :guilabel:`Retention Analysis` page
+is an additional drop-down menu containing different time periods. The default time period is
+:guilabel:`Month`.
+
+When clicked, a drop-down menu of various time period options become available.
+
+.. image:: reports/subscriptions-retention-analysis-time-periods.png
+  :align: center
+  :alt: The time period drop-down menu of the Retention Analysis page.
+
+The time period options are:
+
+- :guilabel:`Day`
+- :guilabel:`Week`
+- :guilabel:`Month`
+- :guilabel:`Year`
+
+When a time period option from this drop-down menu is selected, the data in the :guilabel:`Retention
+Analysis` reporting page showcases data for the configured measures and filters within that time
+period.
+
+To the right of the time period drop-down menu, there is a download button, which allows the user to
+download the data presented on the :guilabel:`Retention Analysis` page as an Excel file.
+
+MRR breakdown
+-------------
+
+To access the :guilabel:`MRR Breakdown` reporting page, navigate to :menuselection:`Subscriptions
+app --> Reporting --> MRR Breakdown`.
+
+By default, the data displayed on the :guilabel:`MRR Breakdown` reporting page is in graph view,
+with the :guilabel:`Bar Chart` option and the :guilabel:`Stacked` option selected.
+
+There is also a default filter in the search bar for :guilabel:`Event Date: Month > Event Type`.
+
+.. image:: reports/subscriptions-mrr-breakdown-default.png
+  :align: center
+  :alt: The default appearance of the MRR Breakdown reporting page in Odoo Subscriptions.
+
+When the :guilabel:`Measures` drop-down menu on the :guilabel:`MRR Breakdown` reporting page is
+clicked, a series of metric-related options become available.
+
+.. image:: reports/subscriptions-mrr-breakdown-measures.png
+  :align: center
+  :alt: The default appearance of the MRR Breakdown reporting page in Odoo Subscriptions.
+
+The metric-related options in the :guilabel:`Measures` drop-down menu on the :guilabel:`MRR
+Breakdown` reporting page are:
+
+- :guilabel:`Active Subscriptions Change`
+- :guilabel:`ARR Change`
+- :guilabel:`MRR Change`
+- :guilabel:`Count`
+
+.. note::
+   The :guilabel:`MRR Change` measure option is selected by default.
```

Comment:
Can we change this if we want, so another one is the default?

Reply:
Yes. If you select a different 'Measure' option, and then click save that search under the "favorites" menu, and click the 'Default filter,' it will be the new default. I'll add that in this note admonition now. Thanks for the suggestion!